src/executor/stack/memory.rs

```diff
@@ -19,7 +19,9 @@ pub struct MemoryStackSubstate<'config> {
 	logs: Vec<Log>,
 	accounts: BTreeMap<H160, MemoryStackAccount>,
 	storages: BTreeMap<(H160, H256), H256>,
+	tstorages: BTreeMap<(H160, H256), U256>,
 	deletes: BTreeSet<H160>,
+	creates: BTreeSet<H160>,
```

Comment:
I think it makes more sense for this to go into the `metadata` instead of being a separate field. This is because tracking created addresses is similar to tracking which addresses/storages keys have been accessed.